### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alajet**
Wanted to add in another machine translation, Alexa Translations A.I.  a leading provider of AI-powered translation solutions for the legal, financial, and government sectors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ This list contains tools, books, articles, blogs, courses and everything related
 
 ### Automatic translation for end users
 - [Bergamot](https://github.com/browsermt) - Client-side translation in the browser.
+- [Alexa Translations](https://https://alexatranslations.com/) - Alexa Translations A.I. brings leading AI and machine translation to the legal, financial and government sectors.
```
Comment (mbiesiad) [inline README.md:71]: Doubled HTTPS protocol (https://) - scheme part (https://en.wikipedia.org/wiki/URL). Please remove redundant one.

**Review by mbiesiad — CHANGES_REQUESTED** [review]
Simple fix needed - URL scheme part

**Comment by alajet** [comment]
Fixed, thank you

**Review by mbiesiad — APPROVED** [review]
LGTM 🚀

**Comment by mbiesiad** [comment]
Hi @alajet Thank you for your effort and contributions here ✨ Great work! Approved, merged 🎉 Best wishes,

**Pull request merged**